Formula/sdl.rb

```diff
@@ -65,7 +65,7 @@ class Sdl < Formula
   end
 
   # SDL 1.2 is deprecated, unsupported, and not recommended for new projects.
-  deprecate! date: "2013-08-17", because: :deprecated_upstream
+  disable! date: "2022-07-28", because: :deprecated_upstream
```

Comment:
`sdl` is the most used deprecated formula. Until we figure out the migration plan to `sdl12-compat`/`sdl2`, this is probably going to stick around.